enterprise/server/util/redisutil/redisutil.go

```diff
@@ -200,26 +209,32 @@ type redlock struct {
 // (b) It does not retry failed locking attempts.
 //
 // See https://redis.io/topics/distlock
```

Comment:
is this comment still correct in light of your changes?

Reply:
yep, it's still using the same underlying implementation (lock=atomically set key to `fixedRandomValue` if not exists; unlock=atomically delete key but only if it's still equal to `fixedRandomValue`), so the limitations are still the same.